cfpb/consumerfinance.gov
# Remove explainer Wagtail CSS

## Removals

- Remove explainer Wagtail CSS


## How to test this PR

1. `python cfgov/manage.py migrate`
2. Visit a form explainer page http://localhost:8000/admin/block-inventory/?include_page_blocks=form_explainer.blocks.ExplainerNote&content_type=
3. See that form explainer blocks are ever so slightly different.


## Screenshots

Before:
<img width="326" alt="Screenshot 2024-09-03 at 2 04 23 PM" src="https://github.com/user-attachments/assets/483d17cb-a1a1-449c-8092-ebcdeafa713a">

After:
<img width="306" alt="Screenshot 2024-09-03 at 2 04 36 PM" src="https://github.com/user-attachments/assets/e052f109-7c87-44d9-8f97-bb0bad472d7c">
